x-pack/elastic-agent/docs/elastic-agent-configuration.asciidoc

```diff
@@ -6,14 +6,17 @@ beta[]
 
 By default {agent} runs in standalone mode to ingest system data and send it to
 a local {es} instance running on port 9200. It uses the demo credentials of the
-`elastic` user. It's also configured to monitor all {beats} managed by the Agent
+`elastic` user. It's also configured to monitor all {beats} managed by {agent}
 and send the {beats} logs and metrics to the same {es} instance.
```

Comment:
If we do a pass through later, we should stop mentioning Beats in most places as these become an implementation detail.

Reply:
I'll change it to "programs" the next time I update the file.